aws-dbt2-ec2/01_deploy/run.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -eux
+
+export ANSIBLE_PIPELINING=true
+export ANSIBLE_SSH_PIPELINING=true
+export ANSIBLE_HOST_KEY_CHECKING=false
+
+python3 ./build-inventory.py "${TERRAFORM_PROJECT_PATH}"
+mv inventory.yml ../.
+
+# FIXME: use absolute path to inventory.yml and vars.yml
+ansible-playbook \
+	-vv \
+	-u "${SSH_USER}" \
+	--private-key "${TERRAFORM_PROJECT_PATH}/ssh-id_rsa" \
+	-i ../inventory.yml \
```

Comment:
we can then remove the need for these variables as each machine will know its user and ssh key path within the inventory file.

```suggestion
	-i "${TERRAFORM_PROJECT_PATH}/inventory.yml" \
```

Reply:
Ok, will remove.